docs/docs/cmd/spp/contentcenter/contentcenter-list.mdx

```diff
@@ -0,0 +1,502 @@
+  ApplyToNewDocumentLibraries | false
+  ArchiveStatus | NotArchived
+  AuthenticationContextLimitedAccess | false
+  AverageResourceUsage | 0
+  BlockDownloadLinksFileType | 0
+  BlockDownloadPolicy | false
+  BlockGuestsAsSiteAdmin | 0
+  BonusDiskQuota | 56244
+  ClearRestrictedAccessControl | false
+  CommentsOnSitePagesDisabled | false
+  CompatibilityLevel | 15
+  ConditionalAccessPolicy | 0
+  CurrentResourceUsage | 0
+  DefaultLinkPermission | 0
+  DefaultLinkToExistingAccess | false
+  DefaultLinkToExistingAccessReset | false
+  DefaultShareLinkRole | 0
+  DefaultShareLinkScope | 0
+  DefaultSharingLinkType | 0
+  DenyAddAndCustomizePages | 2
+  DisableAppViews | 0
+  DisableCompanyWideSharingLinks | 0
+  DisableFlows | 0
+  EnableAutoExpirationVersionTrim | false
+  ExcludeBlockDownloadPolicySiteOwners | false
+  ExpireVersionsAfterDays | 0
+  ExternalUserExpirationInDays | 0
+  GroupId | /Guid(00000000-0000-0000-0000-000000000000)/
+  HasHolds | false
+  HubSiteId | /Guid(00000000-0000-0000-0000-000000000000)/
+  InheritVersionPolicyFromTenant | false
+  IsGroupOwnerSiteAdmin | false
+  IsHubSite | false
+  IsTeamsChannelConnected | false
+  IsTeamsConnected | false
+  LastContentModifiedDate | /Date(2024,6,27,3,35,41,860)/
+  Lcid | 1033
+  LimitedAccessFileType | 0
+  ListsShowHeaderAndNavigation | false
+  LockReason | 0
+  LockState | Unlock
+  LoopDefaultSharingLinkRole | 0
+  LoopDefaultSharingLinkScope | 0
+  MajorVersionLimit | 0
+  MajorWithMinorVersionsLimit | 0
+  MediaTranscription | 0
+  OverrideBlockUserInfoVisibility | 0
+  OverrideSharingCapability | false
+  OverrideTenantAnonymousLinkExpirationPolicy | false
+  OverrideTenantExternalUserExpirationPolicy | false
+  Owner | user@contoso.onmicrosoft.com
+  PWAEnabled | 1
+  ReadOnlyAccessPolicy | false
+  ReadOnlyForBlockDownloadPolicy | false
+  ReadOnlyForUnmanagedDevices | false
+  RelatedGroupId | /Guid(00000000-0000-0000-0000-000000000000)/
+  RequestFilesLinkEnabled | false
+  RequestFilesLinkExpirationInDays | 0
+  RestrictContentOrgWideSearch | false
+  RestrictedAccessControl | false
+  RestrictedToRegion | 3
+  SandboxedCodeActivationCapability | 0
+  SensitivityLabel | /Guid(00000000-0000-0000-0000-000000000000)/
+  SetOwnerWithoutUpdatingSecondaryAdmin | false
+  SharingCapability | 0
+  SharingDomainRestrictionMode | 0
+  SharingLockDownCanBeCleared | false
+  SharingLockDownEnabled | false
+  ShowPeoplePickerSuggestionsForGuestUsers | false
+  SiteDefinedSharingCapability | 0
+  SiteId | /Guid(dcbb0def-d4c3-48f2-aa48-a73a49669491)/
+  SocialBarOnSitePagesDisabled | false
+  Status | Active
+  StorageMaximumLevel | 26214400
+  StorageUsage | 67
+  StorageWarningLevel | 25574400
+  TeamsChannelType | 0
+  Template | CONTENTCTR#0
+  TimeZoneId | 13
+  Title | ContentCenter
+  Url | https://contoso.sharepoint.com/sites/ContentCenter
+  UserCodeMaximumLevel | 0
+  UserCodeWarningLevel | 0
+  WebsCount | 0
+  ```
+
+  </TabItem>
+</Tabs>
```

Comment:
Let's remove this and just import the <Global />

Reply:
Could you confirm what should I import here?